### PR TITLE
Is it tuesday afternoon?  Thought this may be a handy addition.

### PR DESCRIPTION
**Pull request by DooGith**
Won't bore with the details.  Posted on the minetest mesecons [Mod?pack?] forum with pictures and all that.  Simply gives the mese block on state a facelift and adds another kind of 6 point cardinal connector block with a little bit of attitude perhaps.  Maybe people will like the squiggly bits.  Who knows...

### CODE REVIEW

**Comment by DooGith** [comment]
Btw, feel free to remove the fluff.  It's just there to brighten your day and let people know exactly where changes have been made.  Peace

**Comment by sfan5** [comment]
I am not sure what you are doing, but this PR is nowhere near an useful contribution.

**Comment by DooGith** [comment]
The last bit there was to clean up an error in the spelling of "grey".  The game seems to dislike those kinds of inconsistencies.

Ahh...  A biased opinion.  Well done sleepy aqua hair.  This minor change has been extremely useful in my constructs.  They are much more affordable than using mese blocks and much more functional than using the 2 point or 4 point connectors which only trained acrobatic professionals should attempt to assemble in a three dimensional pattern.  Besides, I really like the way they look when activated.  I don't grant myself stacks of 99's, not even on my home world.  "Earning" the materials will always be an important part of the minetest experience to me.  Taking such shortcuts will only serve to degrade said experience.  New ideas and/or new takes on existing ieas.  What's wrong with that?

And if this tiny spit of a non-contribution is cast away.  So be it.  What is it to me if nobody else has the level of enjoyment I get from building on into the future.

I offer these things asking for nothing in return.

Well...  Almost nothing, let's build something!  :)  Peace

**Comment by DooGith** [comment]
I've been trying to understand your earlier comment sfan5.  Perhaps this isn't the place for making such comments as that.  What determines a "useful contribution"?  If a change makes a game look better, work better, behave better, function better how is it not a "useful contribution"?  That's just a single additional 6-point connector block which increases availability, stability and functionality of the mesecon mod.  The grey lightstones have been broken for god only knows how long, the very easy fix of spelling grey "grey" instead of mixing up the colour "grey" with someones name "Gray" fixes the issue and now the nodes are working perfectly every time.  How is this also not a "useful contribution"?  The white lightstone is a direct copy for the mesecon lamp.  It looks like a "mesecon lamp" not a "lightstone" both branches have their own distinctive look and feel.  Changing the white lightstone is only a tiny matter, but it makes the game better by proxy through continuity.  How is this not a "useful contribution"?  Have you actually used a mese block in a mesecon circuit?  It looks horrific, I can take better pictures of a...  I'll stop there.  Should the Mese block not have a good  looking graphic and seem energized when active?  How is fixing that problem not a "useful contribution"?  If all these things added up makes life a little easier, a little more interesting and a little more enjoyable for gamers around the world who appreciate all the love that has been poured into minetest and all the mods to a fraction of nothing.  Wouldn't even that tiny fraction still be considered "quite useful" to those who love the game.

I'm not here to sway anyone's opinion.  I just like to understand the things that confuse me.  Be well, do as you will, I'll speak no more of this.

**Comment by ssieb** [comment]
The problem is your commits.  The commit messages are useless and I expect the comments you've added to the code are not going to be welcome either.  Squash your commits and put useful commit messages.  Also provide a better explanation of what you're trying to do.  And maybe add a small screenshot.  I would also suggest that you add this as an extra option instead of replacing the existing mese block.  I'm pretty sure it will not be accepted otherwise.

File: mesecons_extrawires/mesewire.lua [hunk 1]
```diff
@@ -16,22 +17,57 @@ minetest.override_item("default:mese", {
...
-minetest.register_node("mesecons_extrawires:mese_powered", powered_def)
+--  Ok ok, get out of the kitchen you, here's the recipe
+minetest.register_craft({
+	output = 'mesecons_extrawires:mese_cobble_off 5',
+	recipe = {
+		{'default:cobble', 'mesecons:wire_00000000_off', 'default:cobble'},
+		{'mesecons:wire_00000000_off', 'default:cobble', 'mesecons:wire_00000000_off'},
+		{'default:cobble', 'mesecons:wire_00000000_off', 'default:cobble'},
+	}
+})
```
Comment (numberZero) [inline mesecons_extrawires/mesewire.lua]: 5 cobble in, 5 cobble out. Correct.
But instead of concrete wire, use `group:mesecon_conductor_craftable` in recipes, please. It is exactly for that.

File: mesecons_extrawires/textures/randomread.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+--  This is just a place holder to inject pixel art to this fork.  
+--  By the way, these pixels are variants of the original default:mese and default:cobble nodes.
+--  Modified by myself using gimp
```
Comment (numberZero) [inline mesecons_extrawires/textures/randomread.txt]: ???

File: mesecons_lightstone/init.lua [hunk 3]
```diff
@@ -62,8 +62,8 @@ end
 mesecon.lightstone_add("red", "dye:red", "jeija_lightstone_red_off.png", "jeija_lightstone_red_on.png", "Red Lightstone")
 mesecon.lightstone_add("green", "dye:green", "jeija_lightstone_green_off.png", "jeija_lightstone_green_on.png", "Green Lightstone")
 mesecon.lightstone_add("blue", "dye:blue", "jeija_lightstone_blue_off.png", "jeija_lightstone_blue_on.png", "Blue Lightstone")
-mesecon.lightstone_add("gray", "dye:grey", "jeija_lightstone_gray_off.png", "jeija_lightstone_gray_on.png", "Grey Lightstone")
-mesecon.lightstone_add("darkgray", "dye:dark_grey", "jeija_lightstone_darkgray_off.png", "jeija_lightstone_darkgray_on.png", "Dark Grey Lightstone")
+mesecon.lightstone_add("grey", "dye:grey", "jeija_lightstone_grey_off.png", "jeija_lightstone_grey_on.png", "Grey Lightstone")
+mesecon.lightstone_add("darkgrey", "dye:dark_grey", "jeija_lightstone_darkgrey_off.png", "jeija_lightstone_darkgrey_on.png", "Dark Grey Lightstone")
```
Comment (numberZero) [inline mesecons_lightstone/init.lua]: Er, what does this have to do with the mese cobble wire?

File: mesecons_extrawires/mesewire.lua [hunk 4]
```diff
@@ -16,22 +17,57 @@ minetest.override_item("default:mese", {
...
-	}},
-	groups = {cracky = 1, not_in_creative_inventory = 1},
-	on_blast = mesecon.on_blastnode,
+	}}
+
+})
+
+--  And now for some insanity
+--  Lets add an Off state CobbleCon here
+minetest.register_node("mesecons_extrawires:mese_cobble_off", {
```
Comment (numberZero) [inline mesecons_extrawires/mesewire.lua]: Note that there is a separate file for each wire kind in `extrawires`.

**Comment by numberZero** [comment]
@ssieb Squashing commits is unnecessary, there is “squash and merge” in GH.
@DooGith Please be brief and precise. Large solid blocks of text are hard to read.

**Comment by sfan5** [comment]
@numberZero 
Squashing is unnecessary if it's just a few additions to an existing commit.
If however someone throws me a pile of 12 commits named variations of "No" or "Meh", I am certainly not going to make the effort of splitting/merging them into useful chunks.

**Comment by DooGith** [comment]
Thank you for the useful feedback.  This doomed pull request and fork will be removed soon.  Make the code your own if you like the ideas.  I know that's not standard O.P, but I'm not here to put my name on a plaque.  I'm terrible at dealing with people and even worse at writing code.  But if I in my ignorance can fix mistakes, make things more appealing and cobble together a node that makes life easier for myself and my computer (six sided nodes without all the size and connection constraints seem to be about 20% less CPU intensive, perhaps even more as I realized that I hadn't replaced 100% of the standard connectors in my construct)...  I have to wonder how many people are put off minetest all together just because they don't even know where to start.  Tomorrow.  I'll erase this fork tomorrow.  Nope, it's not Tuesday afternoon yet.  Lol!

**Comment by numberZero** [comment]
@DoogMitt google for `meseconductors` (IIRC), that mod may fit your
needs: it provides conductive cobble, dirt, wood, and other materials.

В Sun, 17 Jun 2018 13:27:43 -0700
DoogMitt <notifications@github.com> пишет:

> Closed #410.
>
